NativeImage.jenkins

```diff
@@ -0,0 +1,144 @@
+pipeline {
+  agent none
+  environment {
+    UPLOAD_LOCATION="localhost" // TODO:
+  }
+  parameters {
+    string (name: 'GRAALVM_VERSION', defaultValue: '20.2.0', description: 'The version of GraalVM to use')
+  }
+  stages {
+    stage("native-image") {
+      parallel {
+        stage("Linux native-image") {
+          agent {
+            label "linux"
```

Comment:
linux||rhel7||rhel8

So it can run on RH infra

Reply:
I missed that it was double pipe the first time, but it should work now.